presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -890,6 +891,11 @@ public SystemSessionProperties(
                         CHECK_ACCESS_CONTROL_ON_UTILIZED_COLUMNS_ONLY,
                         "Apply access control rules on only those columns that are required to produce the query output",
                         featuresConfig.isCheckAccessControlOnUtilizedColumnsOnly(),
+                        false),
+                booleanProperty(
+                        ALLOW_WINDOW_ORDER_BY_LITERALS,
+                        "Skip redundant sort operations",
```

Comment:
I think the description is not accurate.

Reply:
Fixed.